extensions/sql-database-projects/src/test/testUtils.ts

```diff
@@ -111,18 +111,13 @@ export async function createDummyFileStructure(test: Mocha.Runnable | undefined,
 	let filePath = path.join(testFolderPath, 'file1.sql');
 	await fs.writeFile(filePath, '');
 	if (createList) {
-		list?.push(Uri.file(testFolderPath));
```

Comment:
Removed folders because folders no longer need to be explicitly added to the project.  Also because this API is getting changed from "throw anything at it" to being more selective.